source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -86,25 +86,13 @@ async def fetch_page(
 
     url = f"{HUB}/crm/v3/objects/{cls.PROPERTY_SEARCH_NAME}"
 
-    if cls.IGNORE_PROPERTY_SEARCH is True:
-        input = {
-        "associations": ",".join(cls.ASSOCIATED_ENTITIES),
-        "limit": 10,  # 50, # Maximum when requesting history. TODO(johnny).
-        }
-        if len(cls.ASSOCIATED_ENTITIES) == 0:
-            del input['associations']
-    else: 
-        properties = await fetch_properties(log, cls, http)
-        property_names = ",".join(p.name for p in properties.results if not p.calculated)
-
-        input = {
-            "associations": ",".join(cls.ASSOCIATED_ENTITIES),
-            "limit": 10,  # 50, # Maximum when requesting history. TODO(johnny).
-            "properties": property_names,
-            "propertiesWithHistory": property_names,
-        }
-        if len(cls.ASSOCIATED_ENTITIES) == 0:
-            del input['associations']
+    input = {
+    "associations": ",".join(cls.ASSOCIATED_ENTITIES),
+    "limit": 10,  # 50, # Maximum when requesting history. TODO(johnny).
+    }
+    if len(cls.ASSOCIATED_ENTITIES) == 0:
+        del input['associations']
+
     if page:
         input["after"] = page
```

Comment:
Is `after` a page counter or a date? I would kind of expect `cutoff` to be used here, without knowing what the API expects.